### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by MikhailArkhipov**
Removed rproj file language service since it has been causing too many side effects. We lose Open With... on rproj files in Windows Explorer but that's a bit mrginal feature. Opening .rproj from inside VS works as usual.

Added custom action to blow VS MEF cache

### CODE REVIEW

**Comment by int19h** [comment]
:+1:

File: src/Package/Impl/Packages/R/RPackage.cs [hunk 1]
```diff
@@ -93,15 +93,6 @@ namespace Microsoft.VisualStudio.R.Packages.R {
     [ProvideCodeExpansionPath(RContentTypeDefinition.LanguageName, "mrs-distributed", @"Snippets\mrs-distributed")]
     [ProvideCodeExpansionPath(RContentTypeDefinition.LanguageName, "mrs-graphics", @"Snippets\mrs-graphics")]
     [ProvideCodeExpansionPath(RContentTypeDefinition.LanguageName, "mrs-transforms", @"Snippets\mrs-transforms")]
-    #region RProj editor factory
-    // Provide editor factory that instead of opening .rproj file in the editor
-    // locates matching .rxproj file, if any, and opens the project instead.
-    [ProvideEditorExtension(typeof(RProjEditorFactory), RContentTypeDefinition.RStudioProjectExtension, 0x32, NameResourceID = 108, ProjectGuid = RGuidList.CpsProjectFactoryGuidString)]
-    [ProvideLanguageExtension(RGuidList.RProjEditorFactoryGuidString, RContentTypeDefinition.RStudioProjectExtension)]
-    [ProvideLanguageService(typeof(RProjLanguageService), RContentTypeDefinition.RProjectName, 108)]
-    [ProvideEditorFactory(typeof(RProjEditorFactory), 200, CommonPhysicalViewAttributes = 0x2, TrustLevel = __VSEDITORTRUSTLEVEL.ETL_AlwaysTrusted)]
```
Comment (AlexanderSher) [inline src/Package/Impl/Packages/R/RPackage.cs]: So we don't support opening rproj files anymore?

Comment (MikhailArkhipov) [inline src/Package/Impl/Packages/R/RPackage.cs]: Not from Windows Explorer. From inside VS they open fine (i.e. Project | Open works).

**Comment by AlexanderSher** [comment]
:+1:

**Pull request merged**